cmd/kube-controller-manager/app/options/options.go

```diff
@@ -158,6 +159,8 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ServiceAccountKeyFile, "service-account-private-key-file", s.ServiceAccountKeyFile, "Filename containing a PEM-encoded private RSA key used to sign service account tokens.")
 	fs.StringVar(&s.ClusterSigningCertFile, "cluster-signing-cert-file", s.ClusterSigningCertFile, "Filename containing a PEM-encoded X509 CA certificate used to issue cluster-scoped certificates")
 	fs.StringVar(&s.ClusterSigningKeyFile, "cluster-signing-key-file", s.ClusterSigningKeyFile, "Filename containing a PEM-encoded RSA or ECDSA private key used to sign cluster-scoped certificates")
+	fs.BoolVar(&s.ApproveAllKubeletCSRs, "experimental-insecure-approve-all-kubelet-csrs", s.ApproveAllKubeletCSRs, "If enabled, the controller-manager will auto approve all CSRs. WARNING: this grants all users with access to the certificates "+
```

Comment:
string flag passing in the group name to auto-approve kubelet requests from, rather than hard-coding the bootstrap group? that lets deployers decide what bootstrap credential to give kubelets, and what group to allow